dapps/marketplace/src/components/_UserProfileCreated.js

```diff
@@ -1,42 +1,66 @@
 import React from 'react'
 import { fbt } from 'fbt-runtime'
+import useIsMobile from 'utils/useMobile'
+import MobileModal from 'components/MobileModal'
 
-const UserProfileCreated = ({ onCompleted }) => (
-  <div className="profile-created">
-    <img src="images/identity/rocket.svg" />
-    <h2 className="mt-3">
-      <fbt desc="UserActivation.congratulations">Congratulations</fbt>
-    </h2>
-    <div>
-      <fbt desc="UserActivation.profileCreated">
-        You&apos;ve successfully created your profile You&apos;re now ready to
-        continue your journey in the Origin Marketplace.
-      </fbt>
-    </div>
-    <div className="actions">
-      <button
-        type="button"
-        onClick={e => {
-          e.preventDefault()
-          if (onCompleted) {
-            onCompleted()
-          }
-        }}
-        className="btn btn-primary mt-5 mb-3"
-        children={fbt('Ok', 'Ok')}
-      />
+const UserProfileCreated = ({ onCompleted }) => {
+  const isMobile = useIsMobile()
+
+  const content = (
+    <div className="profile-created">
+      <img src="images/identity/rocket.svg" />
+      <h2 className="mt-3">
+        <fbt desc="UserActivation.congratulations">Congratulations</fbt>
+      </h2>
+      <div>
+        <fbt desc="UserActivation.profileCreated">
+          You&apos;ve successfully created your profile You&apos;re now ready to
+          continue your journey in the Origin Marketplace.
+        </fbt>
+      </div>
+      <div className="actions mt-auto">
+        <button
+          type="button"
+          onClick={e => {
+            e.preventDefault()
+            if (onCompleted) {
+              onCompleted()
+            }
+          }}
+          className="btn btn-primary btn-rounded mt-5 mb-3"
+          children={fbt('Ok', 'Ok')}
+        />
+      </div>
     </div>
-  </div>
-)
+  )
+
+  if (isMobile) {
+    return (
+      <>
```

Comment:
no need for the extra `<>`?

Reply:
Yeah, that fragment is not necessary. Will remove it.